.github/workflows/ci.yml

```diff
@@ -111,11 +81,87 @@ jobs:
           docker run --rm --platform=linux/s390x s390x/ubuntu apt update
           docker run --rm --platform=linux/ppc64le ppc64le/ubuntu apt update
           docker run --rm --platform=linux/arm64 arm64v8/ubuntu apt update
-      - name: Test Syscalls
+      -
+        name: Test Syscalls
         if: matrix.target == 'mainline'
         run: |
           set -x
           docker buildx build --platform=linux/amd64,linux/arm64,linux/386,linux/arm,linux/ppc64le,linux/s390x --target=run --allow security.insecure --build-arg CONFIG_RT_GROUP_SCHED=false ./test
+
+  e2e:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        dockerfile:
+          - pip-youtube-dl
+          - webpack
+        platform:
+          - linux/arm/v7
```

Comment:
why arm/v7?

Reply:
It was the platform that failed for the current dockerfiles we have in this PR. If we have more cases in the future with different platforms/inputs we could remove `platform` in the matrix and use a static file with parameters like `docker buildx build $(cat flags) .`?